README.md

```diff
@@ -47,7 +47,7 @@ to your Package.swift-
 
 ```swift
 dependencies: [
-    .package(url: "https://github.com/amzn/smoke-aws.git", .upToNextMajor(from: "1.0.0"))
+    .package(URL: "https://github.com/amzn/smoke-aws.git", .upToNextMajor(from: "1.0.0"))
```

Comment:
Is this change correct? I don't see upper case URL used anywhere in the SwiftPM documentation.